int16 status
int16 distance
float64 left_area
float64 right_area
float64 area_difference 
